packages/sdk/src/base/items.ts

```diff
@@ -13,20 +13,22 @@ export class ItemsHandler<T extends Item> implements IItems<T> {
 		this.endpoint = collection.startsWith('directus_') ? `/${collection.substring(9)}` : `/items/${collection}`;
 	}
 
-	async readOne(id: ID, query?: QueryOne<T>): Promise<OneItem<T>> {
+	async readOne(id: ID, query?: QueryOne<T>, options?: TransportRequestOptions): Promise<OneItem<T>> {
```

Comment:
Instead of passing `TransportRequestOptions` directly into third argument, can you create a new type like `Options` which is basically `{ requestOptions: TransportRequestOptions }`. The same applies to all modifications.

This is intended to allow more options if we need to in the future.

Reply:
Makes sense 👍🏻  I changed it but called the type `ItemsOptions` (will probably have to think about the name one more time) instead of `Options` to reduce the risk that the exported type conflicts with another exported member.